src/core/demangle.d

```diff
@@ -923,7 +923,6 @@ pure @safe:
             return dst[beg .. len];
         case 'F': case 'U': case 'W': case 'V': case 'R': // TypeFunction
             return parseTypeFunction( name );
-        case 'I': // TypeIdent (I LName)
```

Comment:
So `TypeIdent` is not currently being used? Do you know what it was supposed to mean?

Reply:
TypeIdentifier is part of the grammar, but it must be resolved to some other entity during semantics. See https://github.com/dlang/dmd/pull/10526/files#diff-ee89b0981cbb7f8ab2412e68a427d52aR100 for other similar "types".